### PR TITLE
Added "Upcoming events" partial into attend page 

### PR DESCRIPTION
**Pull request by FlorisSp**
screenshot from 2017-03-30 12-28-44

### CODE REVIEW

File: app/views/shared/_upcoming_events.html.erb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<div class="container">
```
Comment (arnoFleming) [inline app/views/shared/_upcoming_events.html.erb]: Fix indentation to improve readabilty

Comment (arnoFleming) [inline app/views/shared/_upcoming_events.html.erb]: in the whole file that is

Comment (arnoFleming) [inline app/views/shared/_upcoming_events.html.erb]: dont mix tabs and spaces, did you copy paste this?

Comment (FlorisSp) [inline app/views/shared/_upcoming_events.html.erb]: I used it from Ewoud, who made something similar for the "Choose your course" section, and adjusted it. Not sure why it shows the weird indentation. It looks fine in atom:

screenshot from 2017-03-30 18-36-08

**Pull request merged**